lib/chef/cookbook/metadata.rb

```diff
@@ -726,7 +617,7 @@ def method_missing(method, *args, &block)
         if block_given?
           super
         else
-          Chef::Log.debug "ignoring method #{method} on cookbook with name #{name}, possible typo or future metadata?"
+          Chef::Log.debug "ignoring method #{method} on cookbook with name #{name}, possible typo or the ghosts of metadata past or future?"
```

Comment:
This puts a smile on my face, but maybe something more informative would be helpful.